src/Theme_Command.php

```diff
@@ -105,7 +105,13 @@ public function status( $args ) {
 	 *
 	 * <search>
 	 * : The string to search for.
-	 *
+     *
```

Comment:
You seem to be using spaces for indentation here, instead of tabs.

WP-CLI uses tabs for indentation and spaces for alignment only.

Reply:
My IDE changed it :/
I will resolve it and send the PR